packages/components/src/vue/vuex-modules/data/lexis.js

```diff
@@ -405,10 +405,6 @@ export default class Lexis extends Module {
       return
     }
 
-    if (!lemmaTranslations && textSelector.languageID === Constants.LANG_LATIN && this._lemmaTranslationLang) {
-      // Use our own rules if lemmaTranslations is not provided
-      lemmaTranslations = { locale: this._lemmaTranslationLang }
-    }
```

Comment:
It is no longer required for the new workflow that do not depend on the `LexicalQuery`.